README.md

```diff
@@ -1,5 +1,5 @@
 # SnapScreen 
-SnapScreen is a project which demonstrates how to use Keycloak, AeroGear Unified Push Server, and OpenShift with React Native.  
+SnapScreen is a project which demonstrates how to use Keycloak, AeroGear Unified Push Server, and OpenShift with React Native. It was created as a demo app for the RedHat summit (2017).  
```

Comment:
I would just change `RedHat summit (2017)` to `Red Hat Summit 2017` to match with how we officially refer to Summit like [here](https://www.redhat.com/en/summit/2018)